server/tools/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@peertube/remote",
+  "version": "1.2.1",
```

Comment:
CLI has its own version (1.0.0 for example)

Reply:
Okay.